src/nt_users_tool/net_commands.py

```diff
@@ -0,0 +1,66 @@
+import os
+
+from typing import NamedTuple
+from constants import LAST_RELEVANT_ELEMENT_POSITION
+
+class NTUserInfo(NamedTuple):
```

Comment:
You can also add a docstring for the class. For is this data structure for? What data does it hold?

Reply:
Okay. Will add.